### DISCUSSION TITLE
Padding only works for specific values

### DISCUSSION THREAD

**Discussion by ukcw** [tailwindlabs/tailwindcss#10624] [Help]
<img width="1721" alt="image" src="https://user-images.githubusercontent.com/34738958/219854354-386995c1-f241-4a4c-9f4c-91746b83b9a6.png">

I have a rather weird problem: when I apply the class `px-4` to line 5 of `About()`, the padding is applied and works as expected. However, any other values for padding e.g. `px-8` does not get applied to the `div`. I have included the directory structure for reference and would appreciate help in figuring this problem :)

Currently, the `About` component's parent is only `body` within `index.tsx`.

**Comment by wongjn** [comment]
Perhaps check the file is covered by the `content` file globs in the Tailwind configuration file.

**Reply by ukcw** [reply]
hi @wongjn, this is my tailwind config file:
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/34738958/219944611-01cce3d8-f95c-43bc-950d-8cab4f8bc019.png">

I think it's correct?

**Reply by wongjn** [reply]
Yes, it seems to be correct. Consider providing a remotely-hosted git repository that reproduces the unexpected behavior.

**Reply by ukcw** [reply]
How do I do this? Do you have a recommended way to do this? I tried using codesandbox.io but the remix app wasn't playing well with their implementation.

**Reply by ukcw** [reply]
As well, my repo is public at https://github.com/ukcw/ukcw.io

**Reply by ukcw** [reply]
@wongjn for some reason, running `npm run build` and working in my dev environment now makes the CSS classes from tailwind work again 😂
